src/features/defi/providers/osmosis/components/OsmosisManager/Withdraw/WithdrawCommon.ts

```diff
@@ -0,0 +1,85 @@
+import type {
+  OsmosisPool,
+  OsmosisToken,
+} from 'state/slices/opportunitiesSlice/resolvers/osmosis/utils'
+import type { LpEarnOpportunityType } from 'state/slices/opportunitiesSlice/types'
+
+type EstimatedFee = {
+  estimatedFeeCrypto?: string
+}
+
+type WithdrawValues = {
+  underlyingAsset0: OsmosisToken
+  underlyingAsset1: OsmosisToken
+  shareInAmount: string
+}
+
+type OsmosisWithdrawValues = WithdrawValues &
+  EstimatedFee & {
+    txStatus: string
+    usedGasFee: string
+  }
+
+export type OsmosisWithdrawState = {
+  opportunity: LpEarnOpportunityType | null
+  poolData: Partial<OsmosisPool> | null
```

Comment:
Not fully aware of the context but will the pool data change in the cycle of a deposit/withdraw ?
I can see `pool_params` contains fees that should be variable, and we also have pool weight and total shares, wondering if this should be re-called in addition to the current call at Deposit/Withdraw modal step?